### PR TITLE
Add strings for Domain Regsitration modal (fx-private-relay/pull/1027)

### PR DESCRIPTION
**Pull request by maxxcrawford**
This is for the domain registration modal feature. 

Screenshot: 

image

### CODE REVIEW

File: en/app.ftl [hunk 1]
```diff
@@ -255,6 +255,17 @@ modal-delete-warning-recovery-html = Once you delete this alias, it cannot be re
 modal-delete-warning-upgrade = If you use this alias to log in to sites you care about, 
     you should update your login with a different email address before you delete this one.
 modal-delete-confirmation = Yes, I want to delete this alias.
+modal-domain-register-good-news = Good News!
+modal-domain-register-warning = Remember you are allowed to register one domain only for your account and is not possible to change your domain later.
```
Comment (peiying2) [inline en/app.ftl:259]: Add a comma after `Remember`

**Review by peiying2 — COMMENTED** [review]
Nit pick on punctuation. Otherwise, it looks good.

File: en/app.ftl (hunk 1 above)
Comment (maxxcrawford) [reply en/app.ftl]: Revised!

**Review by peiying2 — APPROVED** [review]
Thank you for the fix. Looks good!

**Pull request merged**